crowd/hooks/useAuthStore.ts

```
@@ -0,0 +1,317 @@
+          set({
+            error: error.response?.data?.error || "Failed to resend OTP",
+            isLoading: false,
+          });
+          throw error;
+        }
+      },
+
+      // Logout
+      logout: () => {
+        // Remove token from API client
+        delete apiClient.defaults.headers.common["Authorization"];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)